Crear Propuesta Tomas Garzon

tomasgarzon opened the pull request:
# Como matar a tu viejo Django y dejar crecer a sus hijitos

# Se trata de una

* [X] Charla/tutorial/charla relámpago
* [ ] Devroom

# Personas que la proponen
Tomas Garzon, Lead Software Developer en OpenExO

# Condiciones

* [ ] *[Sólo para devrooms]* Acepto coordinarme con la organización de esLibre.
* [ ] *[Sólo para devrooms]* Entiendo que si no hay un programa terminado para la fecha que establezca la organización, la *devroom* podría retirarse.
* [X] Acepto seguir el [código de conducta](https://eslib.re/2019/conducta/) y solicitar a los asistentes y ponentes esta aceptación.
* [X] Al menos una persona entre los que la proponen estará presente en Granada el día 21 junio de 2019.

tomasgarzon committed:
Crear Propuesta Tomas Garzon

JJ reviewed (approved):
Estaría genial si la nueva plataforma se liberara, pero el enfoque de arquitectura de software puede ser interesante de todas formas.

tomasgarzon commented:
Hola, sí estamos liberando la mayor parte del código que tiene sentido que sea reutilizado, ya que no incluye lógica propia de nuestro negocio. Otras veces, estamos usando directamente software libre, como Socketshark o LightBus. En nuestro repo de github podrás encontrar muchos paquetes de los que utilizamos, así como en pypi.

JJ commented:
@tomasgarzon ¿te importaría mucho añadir eso a la propuesta? ¡Gracias!

tomasgarzon committed:
Referencias a software libre

He añadido referencia a las librerías de software libre que estamos utilizando y a las que hemos liberado a traves de nuestro repo y Pypi.

germaaan reviewed (approved):
Lo veo bastante interesante, la explicación del proceso de migración.

germaaan commented:
Según los criterios decididos, con 2 revisiones positivas pasa el corte de aceptación. Gracias!

jsmanrique commented:
La propuesta parece interesante, pero más desde el punto de vista de una empresa contado un business case que desde el punto de vista tecnológico. Parece más la charla de una empresa vendiendo sus capacidades. 
Por otro lado, comentarios como "matar" y el tono despectivo de "viejo" puede generar tensiones innecesarias en la comunidad.

tomasgarzon commented:
Hola, gracias por el feedback.
Respecto al título no hay problema en cambiar, no quería hacer referencia a
Django monolítico y servicios, ya que es algo más común, pero si puede
generar controversia busco otra propuesta.
Respecto al business Case, creo que no se ha entendido lo que quiero
presentar. Es una charla muy técnica de cómo transformar un django
monolitico desarrollado durante años en una arquitectura más flexible,
tanto desde el punto de vista del desarrollador como desde el staff en la
búsqueda del modelo de negocio. Son lecciones que hemos aprendido en este
proceso de cambio, y nuevos retos que aún tenemos pendientes de solucionar.
Quedo a tu disposición por si necesitas alguna aclaración más.

On Thu, Apr 18, 2019, 10:55 AM Manrique Lopez <notifications@github.com>
wrote:

> La propuesta parece interesante, pero más desde el punto de vista de una
> empresa contado un business case que desde el punto de vista tecnológico.
> Parece más la charla de una empresa vendiendo sus capacidades.
> Por otro lado, comentarios como "matar" y el tono despectivo de "viejo"
> puede generar tensiones innecesarias en la comunidad.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslibre/charlas/pull/26#issuecomment-484414812>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAOLFMBSWBGSCRHURDOBAC3PRAZQTANCNFSM4HGAHEFA>
> .
>
